formulador-lite.asd

```diff
@@ -0,0 +1,26 @@
+;;;;formulador-lite.asd
+;;;;
+;;;;Copyright (c) 2021 Izaak Walton
+
+(defpackage #:formulador-lite-asd
+  (:use #:cl))
+
+(in-package #:formulador-lite-asd)
+
+(asdf:defsystem #:formulador-lite
+  :author "Izaak Walton <izaakw@protonmail.com>"
+  :license "BSD 3-clause (see LICENSE)"
+  :description "Simplified formula interface."
+  :long-description "A simple input interface for Formulador."
+  :depends-on (#:formulador #:alexa)
+  :serial t
+  :components ((:module "formulador-lite"
+		:serial t
```

Comment:
indentation wonky due to tabs?

Reply:
I've fixed the formatting, just figuring out evaluation issues- is it common practice to "Resolve conversation" as I fix stuff, or to leave it until I push an updated request?